### PR TITLE
DOC: Example notebook with diagnostic plots useful to identify potential problem in linear regression

### PR DESCRIPTION
**Pull request by pkaf**
Example notebook to visualize potential problems in linear regression.

Diagnostic presented in the notebook are sort of first few plots usually made in order to identify, if any, potential problem in fitting linear regression.

### CODE REVIEW

**Comment by bashtage** [comment]
You shouldn't submit the .py version.  This version will be auto-created using a script later from the ipynb.

**Comment by pkaf** [comment]
My bad. Shall I remove the file and push the changes again? 

 Also, I think the instruction confused me. Seems text after second sentence is redundant, do you reckon? 
Screen Shot 2022-02-07 at 8 04 31 pm

**Comment by bashtage** [comment]
Those instructions are out of date, sorry.  You should remove the .py file, than you can rebase and squash to a single commit.

**Comment by bashtage** [comment]
Also can you uncommit the png.  We only commit unexecuted notebooks, and then our doc builder runs the notebook.

File: examples/notebooks/linear_regression_diagnostics_plots.ipynb [hunk 1]
```diff
@@ -0,0 +1,587 @@
...
...TUto/WXJMDgFtFZDVwEXCViFyQ5PS2l9Zcl3XAElXdqqplwAvAvklPcfK15pr8BHhRVUOquhl4A9gj+/JJZeB/FxgpIsNEJANXyfJsnW2eBc7wvv8IeE1dzcuzwMle7fwwYCTwf+2U7mRr8XURkXxgAXClqr7RXgluBy2+Jqp6sKoOVdWhwB3ATar6/9op3cnWmv9DC4F9RCTHC36HAh+3U7qTqTXXZA1wOICIdAEOBD5pl1S3t1TWLANHA5/hauGv9pZdDxznfc/CtcT4AhfYh8fte7W336fAd1NdS94RrgvwW1wZ5ftxU+9U/55U/63EHWMOe1CrntZeF+A0XIX3SuDWVP+WVF8TINdb/hEuE7ws1b8lWZN12WCMMZ2MvblrjDGdjAV+Y4zpZCzwG2NMJ2OB3xhjOhkL/MYY08lY4DedgoiUpDoNxnQUFviNaUPey1DGdGgW+E2nJSIjRORFb+yC10VkbxHJE5FvRMTnbdNFRNZ6fbXvtr23zYMicreIvIPrHmJ/r5/75SLypoiM9rbLEZEnRORjEZnv9QU/1Vv3HW+f90TkSa9zOWOSwu5OTGd2DzBbVT8XkQOAu1T1cBF5H9eFwSLgWGChqoZEZLft8V7xx/UJc5CqRkSkG3CwqoZF5AjgJlxvqecBO1R1rIiMx71ZjYj0wr11fYSqlorIFcAluLdNjWlzFvhNp+TdUR8EPBnXa2ds7ILHgZNwgf9k4K4mtgd4Umv6s88D5orISFwf70Fv+beBPwGo6koRWeEtPxA3uNAb3rEzgLfa5pcaszsL/Kaz8gE7VXVSPeueBW4SkR7AFOA1oEsj24PrIynmBmCRqh7vDeu3uIm0CPCyqp6ScOqNaQUr4zedkqruAr4WkR8DiDPRW1eC6+XxT8DzqhppbPt65FHTFfCZccvfAE709h8L7OMtfxv4lojs5a3rIiKj2uaXGrM7C/yms8gRkXVx0yXAqcDZIvIBrkfG+CH6Hsf1Xvl43LLGto93K3CziCyn9lP1XUCBiHwM3Ogdo0hVt+AyiH96xT9vAXu37uca0zDrndOYdiIifiCoqhUiMgJ4BRitbmxYY9qNlfEb035ygEUiEsSV659nQd+kgt3xG2NMJ2Nl/MYY08lY4DfGmE7GAr8xxnQyFviNMaaTscBvjDGdzP8HsaEeMc0PHuYAAAAASUVORK5CYII=",
```
Comment (bashtage) [inline examples/notebooks/linear_regression_diagnostics_plots.ipynb]: Please clean the notebook and remove all executed cells.

**Pull request closed without merging**